k2/csrc/array_test.cu

```diff
@@ -297,6 +297,87 @@ void TestArray2() {
     context = GetCudaContext();
   }
 
+  {
+    // test To(context)
+    // case 1: contiguous
+    //
+    // 0 1 2
+    // 3 4 5
+    //
+    constexpr auto kDim0 = 2;
+    constexpr auto kDim1 = 3;
+    std::vector<T> data = {0, 1, 2, 3, 4, 5};
+    ASSERT_EQ(static_cast<int32_t>(data.size()), kDim0 * kDim1);
+
+    Array1<T> arr1(context, data);
+    Array2<T> array(arr1, kDim0, kDim1);
+
+    auto cpu_array = array.To(cpu);
+    auto cuda_array = array.To(GetCudaContext());
+
+    ASSERT_EQ(cpu_array.ElemStride0(), cpu_array.Dim1());
+    ASSERT_EQ(cuda_array.ElemStride0(), cuda_array.Dim1());
+
+    auto k = 0;
+    for (auto r = 0; r != kDim0; ++r)
+      for (auto c = 0; c != kDim1; ++c) {
+        // WARNING: it's inefficient to access elements of Array2
+        // with operator [][]
+        EXPECT_EQ(cpu_array[r][c], k);
+        EXPECT_EQ(cuda_array[r][c], k);
+        ++k;
+      }
+
+    // test operator <<
+    std::ostringstream os;
+    os << array;
+    K2_LOG(INFO) << os.str().c_str();
```

Comment:
Just use` K2_LOG(INFO) << array`, I redefine template function in `Logger `

https://github.com/danpovey/k2/blob/master/k2/csrc/log.h#L113-L118

Reply:
Done.